soot-infoflow/src/soot/jimple/infoflow/results/xml/InfoflowResultsSerializer.java

```diff
@@ -188,6 +189,12 @@ protected void writeDataFlows(InfoflowResults results, XMLStreamWriter writer) t
 
 	}
 
+	private String getMethodSignature(MethodSourceSinkDefinition smac) {
```

Comment:
Wouldn't it make more sense to not write the attribute rather than writing some human-readable but totally artificial string into the attribute? I guess parsing the file can get very ugly if we need to check for such well-known strings.